linux/help/reference/index.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
We shouldn't be including .php files in the keyman repo

Reply:
@ermshiperete - I'll apply your suggestions to the php files in the help.keyman.com repo